docs/_news/news06.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: news
+title: New for Codewind 0.6.0
+description: New for Codewind 0.6.0
+keywords: news, new, updates, update, version
+duration: 1 minute
+permalink: news06
+---
+
+## Codewind 0.6.0
+Thursday 21 November 2019
+
+#### Codewind workspace updates
+**VS Code**
+- You no longer need to place projects in the `codewind-workspace`.
+- Do not create projects in the `~/codewind-data/` or `C:\codewind-data` directories. User configuration files are now kept in `~/.codewind/` or `C:\Users\<user>\.codewind\`.
+- If you want to move an existing project, remove it from Codewind without deleting the files. Then, run the `Add Existing Project` command to reintroduce it to its new location.
```

Comment:
Should probably say something like  `...remove it from Codewind without deleting the files.  Next, move the project to it's new location.  Then, run the 'Add Existing Project' command to reintroduce it from its new location.`